File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodesResult.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodesResult (
    _searchId: String,
    _postcodes: List[ResponseTimeFilterPostcode]
)
object ResponseTimeFilterPostcodesResult {
    def toStringBody(var_searchId: Object, var_postcodes: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"postcodes":$var_postcodes
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseTimeFilterPostcodesResult.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseTimeFilterPostcode

module ResponseTimeFilterPostcodesResult = 

  //#region ResponseTimeFilterPostcodesResult


  type ResponseTimeFilterPostcodesResult = {
    SearchId : string;
    Postcodes : ResponseTimeFilterPostcode[];
  }
  //#endregion
  